simple_type_field_names.json5

```diff
@@ -80,6 +84,13 @@
 
   // Entity
   "net/minecraft/unmapped/C_rjqjaxef": "brain",
+  "net/minecraft/unmapped/C_jzrpycqo": "player", // Player
+  "net/minecraft/unmapped/C_jzrpycqo": { // PlayerEntity
```

Comment:
Duplicate key

Reply:
I'd also add its variants (Client and Server ones)

Reply:
Ok how did I even do that :sob: